tools/topology/CMakeLists.txt

```diff
@@ -73,6 +73,7 @@ set(TPLGS
 	"sof-apl-da7219\;sof-apl-da7219"
 	"sof-glk-da7219-kwd\;sof-glk-da7219-kwd\;-DPLATFORM=glk"
 	"sof-glk-da7219-kwd\;sof-cml-da7219-max98357a\;-DPLATFORM=cml"
+	"sof-glk-da7219-kwd\;sof-cml-da7219-max98390\;-DCODEC=MAX98390\;-DPLATFORM=cml"
```

Comment:
this file is not known to the kernel, is there any plan to upstream the matching changes for Linux?

Reply:
@plbossart Thanks for notifying! I upstream the code changes for your reviewing. https://patchwork.kernel.org/project/alsa-devel/patch/1616757689-22014-1-git-send-email-mac.chiang@intel.com/